src/aleph/db/accessors/messages.py

```diff
@@ -87,7 +89,15 @@ def make_matching_messages_query(
         select_stmt = select_stmt.where(MessageDb.sender.in_(addresses))
     if chains:
         select_stmt = select_stmt.where(MessageDb.chain.in_(chains))
+    if message_types:
+        if len(message_types) == 1:
+            select_stmt = select_stmt.where(MessageDb.type == message_types[0])
+        else:
+            select_stmt = select_stmt.where(MessageDb.type.in_(message_types))
```

Comment:
Just use `in_`, we didn't measure if there was a performance difference so let's keep it simple. I'd be surprised if there's one.